packages/util/src/Classification.ts

```diff
@@ -0,0 +1,6 @@
+import { PCD } from "@pcd/pcd-types";
+import _ from "lodash";
+
+export function matchPCDFields(fields: Partial<PCD>, pcd: PCD): boolean {
```

Comment:
what's the purpose of this function? i can sort of visualize what it's doing, but why might one want to do this?